novawallet/Modules/Vote/Governance/Operation/Fetch/OffchainVoting/SubqueryVotingOperationFactory.swift

```diff
@@ -149,10 +168,16 @@ extension SubqueryVotingOperationFactory: GovernanceOffchainVotingFactoryProtoco
 
     func createReferendumVotesFetchOperation(
         referendumId: ReferendumIdLocal,
-        isAye: Bool
-    ) ->
-        CompoundOperationWrapper<[ReferendumVoterLocal]> {
-        let query = prepareReferendumVotersQuery(referendumId: referendumId, isAye: isAye)
+        votersType: ReferendumVotersType
+    ) -> CompoundOperationWrapper<[ReferendumVoterLocal]> {
+        let query = switch votersType {
+        case .ayes:
+            prepareReferendumVotersQuery(referendumId: referendumId, isAye: true)
+        case .nays:
+            prepareReferendumVotersQuery(referendumId: referendumId, isAye: false)
+        case .abstains:
+            prepareSplitAbstainVotesQuery(referendumId: referendumId)
```

Comment:
I would have a separate request for splitAbstain votes as it will have different model